### PR TITLE
Cleanup: WoA Logging feature flags

### PR DESCRIPTION
**Pull request by zaguiini**
## Proposed Changes

It's time to remove the flags introduced while working on WoA Logging i1.

## Testing Instructions

Check that `/site-logs/%s` is still available and that the logs card on `/hosting-configuration/%s` is gone.

The "Web server logs" entry on the site ellipsis menu on `/sites` should not be seen.

Finally, the "Site logs" entry should show up.

### CODE REVIEW

**Review by mpkelly — APPROVED** [review]
Tested on the branch and everything works as expected. Checked for `woa-logging` references in the code but didn't find any. LGTM

**Comment by mpkelly** [comment]
The test is fixed, so I am going to merge this.

**Comment by zaguiini** [comment]
Ok @mpkelly I was wondering if Philip would have additional thoughts since he was the main person working on it, but he can revert the PR if something weird happens.

**Comment by mpkelly** [comment]
@zaguiini then you need to add the `Philip Can Merge` label 😉

**Comment by zaguiini** [comment]
I might go ahead and create additional labels for every Automattician then @mpkelly 🤣

**Comment by p-jackson** [comment]
Change looks good and `/site-logs/{{ simple site slug }}` is still redirecting to `/home/{{ simple site slug }}` as expected.

> then you need to add the `Philip Can Merge` label 😉

Please no 🙏 lol

I feel like if we want a specific person to look at a PR we should leave off the `[Reviewer Can Merge]` label. Feels similar to the situation when you'd rather merge the PR yourself. Except you can ping the person you want a review from directly.

File: config/development.json [hunk 1]
```diff
@@ -205,8 +205,6 @@
 		"wordpress-action-search": false,
 		"wpcom-user-bootstrap": false,
 		"newsletter/stats": true,
-		"newsletter/paid-subscribers": true,
-		"woa-logging": true,
-		"woa-logging-moved": true
+		"newsletter/paid-subscribers": true
```
Comment (klimeryk) [inline config/development.json:208]: Please keep this list in alphabetical order (and other config files in `/config`) as per the [README file here](https://github.com/Automattic/wp-calypso/blob/dbbed47493fca010895952775bd0852868e7d23e/config/README.md#feature-flags):

> Please make sure to add new feature flags alphabetically so they are easy to find

🙏

Comment (p-jackson) [reply config/development.json:208]: Tidied in 674a4fdd3b14ffbe28c96948f2dd9b65521ddeef CC @JuanLucha

Comment (klimeryk) [reply config/development.json:208]: Awesome, appreciate it! 🙇

**Pull request merged**